reactjs-todo/e2e/virtual-authenticator.spec.js

```diff
@@ -0,0 +1,32 @@
+import { test, expect } from '@playwright/test';
+
+test('webAuth login react', async ({ browser, page }) => {
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
```

Comment:
Rather than using if statements like this, can we use playwright filters

https://playwright.dev/docs/test-projects#splitting-tests-into-projects 

https://playwright.dev/docs/test-projects#test-filtering

We should be able to leverage this by scoping tests that require certain "projects".

For example:

If we follow a testing name pattern like this

[Framework] [Action] [Feature] we can generate a title like

React - Login with WebAuthn

This allows us to A) filter tests by `/React/` , `/WebAuthn/` and `/login`/

This will give us flexibility in runnings tests for regressions, but also a standard for writing test names.

@juangarmendia05 @cerebrl let me know your thoughts.

Reply:
Yeah, I'm good with this convention. Are you suggesting we make this change now or in a future update.

Reply:
We can do it later if we want, i think its a fairly quick update though.